src/hww.c

```diff
@@ -67,6 +70,52 @@ static void _api_attestation(const Packet* in_packet, Packet* out_packet)
     memcpy(data, result.challenge_signature, sizeof(result.challenge_signature));
 }
 
+/**
+ * Serializes sytem information to the buffer. The first 12 bytes are considered stable to extract
+ * the version and should never change.
+ * The following bytes are written:
+ * 12 bytes: short firmware version string, ascii encoded, right-padded with 0x00. E.g. "v4.12.2".
+ * 1 byte: platform code:
+ * - 0x00 - BitBox02
+ * - 0x01 - BitBoxBase
+ * 1 byte: edition code:
+ * - For the BitBox02 edition:
+ * - - 0x00 - Multi
+ * - - 0x01 - Bitcoin-only
+ * - For the BitBoxBase platform:
+ " - 0x00 - Standard
+ * 1 byte: 0x00 if the device is locked, 0x01 if the device is unlocked.
+ * @param[out] buf serialize info to this buffer.
+ * @return number of bytes written
+ */
+static size_t _api_info(uint8_t* buf)
+{
+    uint8_t* current = buf;
+    { // 12 bytes version string, 0x00-padded
```

Comment:
I don't think the introduced scope here helps with anything. There are no local variables. Same applies below.

Reply:
done. even with no local vars i did it so it's visually easy to make out the individual fields in the response.